app/src/main/java/com/simplecity/amp_library/model/Artist.java

```diff
@@ -21,10 +21,7 @@ public boolean equals(Object o) {
 
         Artist artist = (Artist) o;
 
-        if (id != artist.id) return false;
-        if (numAlbums != artist.numAlbums) return false;
-        if (numSongs != artist.numSongs) return false;
-        return name != null ? name.equals(artist.name) : artist.name == null;
+        return id == artist.id && numAlbums == artist.numAlbums && numSongs == artist.numSongs && (name != null ? name.equals(artist.name) : artist.name == null);
```

Comment:
I'd prefer not to make this change. 'Simplifying' the conditional statement generally makes it harder to read for equals() method.